plugins/flutter_aepedge/lib/src/aepedge_eventhandle.dart

```diff
@@ -0,0 +1,37 @@
+/*
+Copyright 2022 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License.
+*/
+
+/// EventHandle is the response returned from Adobe Experience Platform Edge Network.
+class EventHandle {
+  EventHandle(this.data);
+
+  EventHandle.createEvent(
+    final String type,
+    final List<dynamic> payload,
+  ) {
+    final Map<dynamic, dynamic> eventHandleConstructorData = {
+      "type": type,
+      "payload": payload
+    };
+    this.data = eventHandleConstructorData;
+  }
+
+  late Map<dynamic, dynamic> data;
```

Comment:
can we add documentation for this variable

Reply:
This data returns as Map<object?, object?> for type, can't use Map<String, dynamic>.  
We had the similar issue for the Consent response.  
![eventhandle](https://user-images.githubusercontent.com/29806547/182562951-fbd3950b-6869-426f-b6e3-5b5335970dc0.png)

Reply:
we resolved this one, will keep dynamic, dynamic as from platform and make it private.